### ISSUE TITLE
The confusion about the code.

### ISSUE THREAD

**Issue by htyj123** [fenglinglwb/MAT#29]
Hello, thank you for your job, I wonder why the masks need to substract the 0.5 in the code "x = torch.cat([masks_in - 0.5, images_in * masks_in], dim=1)" of the firststage network.

**Comment by fenglinglwb** [comment]
Thanks for your attention to our work. You may use 'masks_in' without subtraction.

**Comment by xiaolai2020** [comment]
Hello, I also want to know why I subtracted 5, and  what's the use of it. Thank you
